net.jockx.Kulki.yml

```diff
@@ -0,0 +1,27 @@
+id: net.jockx.Kulki
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: Kulki
+finish-args:
+  - --socket=x11
+  - --share=ipc
+modules:
+  - name: kulki
+    buildsystem: simple
+    build-commands:
+      - cp -r bin/. $FLATPAK_DEST/bin
+      - cp -r lib/. $FLATPAK_DEST/lib
+      - install -Dm644 "${FLATPAK_ID}.desktop" -t "/app/share/applications"
+      - install -Dm644 "${FLATPAK_ID}.png" -t "/app/share/icons/hicolor/512x512/apps/"
+      - install -Dm644 "${FLATPAK_ID}.metainfo.xml" -t "${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml"
+    sources:
+      - type: archive
+        url: https://github.com/jockx-net/kulki/releases/download/v1.0.0/Kulki-1.0.0.tar.gz
```

Comment:
how will this work on multiple architectures? I know that JAVA is compile once debug everywhere, but the JDK surely ain't.

Reply:
This app is not multiplaform. It's a amd64 release only at this point. Should I specifiy that somewhere explicitly?

Reply:
so it builds a broken aarch64 flatpak.

Reply:
You need a flathub.json file https://docs.flathub.org/docs/for-app-authors/maintenance#limiting-the-set-of-architectures-to-build-on